src/targets/browser/index.ejs

```diff
@@ -11,6 +11,8 @@
   <meta name="msapplication-TileColor" content="#2b5797">
   <meta name="theme-color" content="#ffffff">
   <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1, viewport-fit=cover">
+  <link rel="preload" href="/assets/fonts/Lato-Bold.immutable.woff2" as="font" type="font/woff2" crossorigin>
+  <link rel="preload" href="/assets/fonts/Lato-Regular.immutable.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
It's asked by MDN citing W3C: https://drafts.csswg.org/css-fonts/#font-fetching-requirements
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload#cors-enabled_fetches

Sure looks strange because it's the same origin but seems it's needed for fonts anyway

Reply:
I didn't know that 🤯 

Should we verify that this works on production stack?